src/redux/reducers/employer.ts

```diff
@@ -11,7 +11,7 @@ const initialState = {
   error: null,
 }
 
-const test = (state = initialState, action: employerActions) => {
+const test = (state = initialState, action: EmployerActions) => {
```

Comment:
You can change the name of this reducer 😄 , I forgot to change it

Reply:
That's just the name of the type of the employer's actions. :))